Source/Core/Core/PowerPC/Jit64/Jit.cpp

```diff
@@ -599,6 +602,13 @@ void Jit64::WriteBLRExit()
   bool disturbed = Cleanup();
   if (disturbed)
     MOV(32, R(RSCRATCH), PPCSTATE(pc));
+  const u64 msr_bits = m_ppc_state.msr.Hex & JitBaseBlockCache::JIT_CACHE_MSR_MASK;
+  if (msr_bits != 0)
+  {
+    MOV(32, R(RSCRATCH2), Imm32(msr_bits));
+    SHL(64, R(RSCRATCH2), Imm8(32));
```

Comment:
Couldn't we use something like:
```
MOV(64, R(RSCRATCH2), Imm64(msr_bits << 32));
```

Or is there some benefit using MOV+SHL?

Reply:
The code is smaller this way. But the difference is just one byte, so I'm not convinced that either way is the best...

Reply:
ok, makes sense.